packages/wonder-blocks-layout/util/test-util.test.js

```diff
@@ -0,0 +1,275 @@
+// @flow
+import {resizeWindow, checkQuery, matchMedia} from "./test-util.js";
+
+describe("Test utils", () => {
+    describe("resizeWindow", () => {
+        it("should set the width to 1200px for large", () => {
+            // Arrange
+            const expectedWidth = 1200;
+
+            // Act
+            resizeWindow("large");
+
+            // Assert
+            expect(window.innerWidth).toBe(expectedWidth);
+        });
+
+        it("should set the width to 800px for medium", () => {
+            // Arrange
+            const expectedWidth = 800;
+
+            // Act
+            resizeWindow("medium");
+
+            // Assert
+            expect(window.innerWidth).toBe(expectedWidth);
+        });
+
+        it("should set the width to 640px for small", () => {
+            // Arrange
+            const expectedWidth = 640;
+
+            // Act
+            resizeWindow("small");
+
+            // Assert
+            expect(window.innerWidth).toBe(expectedWidth);
+        });
+
+        it("should dispatch a 'resize' event", async () => {
+            // Arrange
+            let handler;
+            const promise = new Promise((resolve, reject) => {
+                handler = window.addEventListener("resize", resolve);
+            });
+
+            // Act
+            resizeWindow("small");
+            await promise;
+
+            // Assert
+            expect(promise).resolves.toHaveProperty("type", "resize");
+            window.removeEventListener("resize", handler);
+        });
+    });
+
+    describe("checkQuery", () => {
+        describe("(max-width: 767px)", () => {
+            // Arrange
+            const queryString = "(max-width: 767px)";
```

Comment:
I don't think this constant is valuable. Rather have the string inline in the test so the test case is easily understood as one thing.